Components/LogoWithName.tsx

```diff
@@ -1,8 +1,9 @@
 import Image from "next/image";
 const LogoWithName = ({ width, height }: any) => {
   return (
-    <div className="flex  items-center   w-full h-fit  justify-center ">
-      <Image src="./logo.svg" width={width} height={height} alt="logo" />
+    <div className="flex  items-center   w-full h-fit  justify-center">
+     <Image width={40} height={40} src="./logo.svg" alt="logo" className="dark:hidden"/>
+          <Image width={40} height={40} src="./logodark.svg" alt="logo" className="hidden dark:block"/>
```

Comment:
How are these two images working?

Reply:
Unfortunately the fill aspect does not work when we use Image component in tailwind, So we have to use 2 seperate Images, 1 for dark mode and 1 for light mode. When we use dark mode, dark:hidden class hides the light mode logo and when we use light mode, the dark mode logo is hidden by the hidden attribute. You can refer tailwind docs for more info.